File: luiz020/programacao_basica/questao14.py
print(" Programa de Tempo de Trabalho ")

dias = int(input("Informe a quantidade de dias: "))

anos = dias // 360

dias = dias - (anos * 360)

meses = dias // 30

dias = dias - (meses * 30)

print("Sem acidentes em anos: " + str(anos))

print("Sem acidentes em meses: " + str(meses))

print("Sem acidentes em dias: " + str(dias))

File: luiz020/programacao_basica/Questao14.java
import java.util.Scanner;

    public class Questao14 {

        public static void main (String[] args) {

        Scanner k = new Scanner(System.in);

        int dias, meses, anos;

        System.out.println(" Programa Sem Acidente no Trabalho ");

        System.out.print("Informe a quantidade de dias: ");
        dias = k.nextInt();

        anos = dias / 360;

        dias = dias - (anos * 360);

        meses = dias / 30;

        dias = dias - (meses * 30);

        System.out.println("Sem acidentes em anos: " + anos); 
        System.out.println("Sem acidentes em meses: " + meses); 
        System.out.println("Sem acidentes em dias: " + dias);

        }}
           
                   
